iot-hub/Samples/service/DeviceStreamingSample/ServiceClientStreamingSample.csproj

```diff
@@ -1,8 +1,9 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
why 2.1 specifically?

Reply:
Just making them all the same, and 2.1 is an LTS.

Reply:
https://dotnet.microsoft.com/download/dotnet-core